cmd/rollapp/init/init.go

```diff
@@ -165,7 +165,13 @@ func Cmd() *cobra.Command {
 					RpcUrl:        chd.RpcUrl,
 					ArchiveRpcUrl: chd.RpcUrl,
 					GasPrice:      chd.GasPrice,
-					DaNetwork:     consts.CelestiaTestnet,
+					// DaNetwork:     consts.CelestiaTestnet,
+					DaNetwork: consts.AvailTestnet,
```

Comment:
please remove occurances where avail is set as the default da to replace celestia, currently it's not tested/integrated into roller and isn't considered stable